setup.ps1

```diff
@@ -1,115 +1,127 @@
 param (
-    [string]$OracleContainerName = "psw-oracle-1",
-    [string]$StorageContainerName = "psworacle1",
-    [string]$ConnectionStringName = "OracleConnectionString",
-    [string]$TagName = "setup-oracle-action",
```

Comment:
I'm not entirely keen on removing the defaults here because they are super handy when you need to test it locally. But since they are only valuable for testing I guess I'm OK to let it go and forcing myself in the future to add all the necessary values when invoking the script